apps/src/templates/instructions/CollapserIcon.jsx

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var _ = require('lodash');
+var color = require('../../color');
+
+var styles = {
+  showHideButton: {
+    position: 'absolute',
+    top: 0,
+    left: 8,
+    margin: 0,
+    lineHeight: '30px',
+    fontSize: 18,
+    // get hover behavior from CollapserIcon_showHideButton
```

Comment:
I'm increasingly leaning towards introducing radium, which allows us to do things like :hover on inline styles. The other neat thing it gets us is the ability to provide an array of styles in jsx, so instead of 
```
var myStyle = _.assign({}, styles.base, {
  display: shouldShow ? 'block' : 'hidden' 
});
...
return <Foo style={myStyle}/>
```
I can do something like
```
return <Foo styles=[styles.base, !shouldShow && styles.hide]/>
```